aten/src/ATen/core/NamedRegistrations.cpp

```diff
@@ -113,10 +113,6 @@ TORCH_LIBRARY_IMPL(aten, Named, m) {
   m.impl("conj.out", CppFunction::makeFallthrough());
   m.impl("contiguous", CppFunction::makeFallthrough());
   m.impl("copy_", CppFunction::makeFallthrough());
-  m.impl("copy_imag", CppFunction::makeFallthrough());
```

Comment:
These 5 fn's all don't exist in native_functions.yaml. I also found this PR implying that copy_[real|imag] were removed at some point: https://github.com/pytorch/pytorch/pull/39065

Reply:
Yes, removing these looks super harmless.